Assets/MixedRealityToolkit-SDK/Features/UX/Materials/UIDarkGray.mat

```diff
@@ -0,0 +1,173 @@
+%YAML 1.1
```

Comment:
Should this be in Standard Assets?

Reply:
Agreed, materials / textures for the SDK should be in the standard assets folder